unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -230,6 +231,8 @@ private void SetVisibility_Internal(bool visible)
             if (DataStore.i.featureFlags.flags.Get().IsFeatureEnabled("backpack_editor_v2"))
                 view.SetSectionAsNew(ExploreSection.Backpack, true);
 
+            view.SetWalletActive(DataStore.i.wallet.isInitialized.Get());
```

Comment:
Here we would have to avoid to use the `DataStore` static instance and inject it in the constructor but, due to the amount of changes that I would have to apply in this class, in order to not make this implementation dirt with unrelated changes, I will annotate it for doing it in a separated PR.